CHANGELOG.d/bug_add-exclude-file-to-more-commands.md

```diff
@@ -0,0 +1,5 @@
+* Add `--exclude-file` to more commands
```

Comment:
`bug_` isn't one of the sacred file name prefixes; you'll want `fix_`. (But maybe this is actually `feature_`?)

Reply:
I think this is the second time I've done this: calling it `bug_` rather than `fix_`. :smile: 
I've renamed it to `feature_`.